File: Client/HextechFriendsClient/Protocol/Server/RequestJoinTC.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HextechFriendsClient.Protocol.Server
{
    public class RequestJoinTC
    {
        public string opCode { get; set; }
        public string uuid { get; set; }
        public string summonerName { get; set; }
        public string summonerId { get; set; }
        public int level { get; set; }
        public int iconId { get; set; }
    }
}

File: Server/src/main/java/xyz/sleepylux/hextechfriends/models/DTO/server/RequestJoinTC.java
package xyz.sleepylux.hextechfriends.models.DTO.server;

public class RequestJoinTC {
    private String opCode = "requestJoinTC";
    private String uuid;
    private String summonerName;
    private String summonerId;
    private int level;
    private int iconId;

    public void setUuid(String uuid) { this.uuid = uuid; }

    public void setLevel(int level) { this.level = level; }

    public void setIconId(int iconId) { this.iconId = iconId; }

    public void setSummonerName(String summonerName){ this.summonerName = summonerName; }

    public void setSummonerId(String summonerId) { this.summonerId = summonerId; }
}
